examples/run_all.py

```diff
@@ -183,19 +188,25 @@ def do_one_mmw(dirname, progname, npyfile, efargstring, mmwargstring):
        "--instance-name=sslp_15_45_10 --bundles-per-rank=2 "
        "--max-iterations=5 --default-rho=1 "
        "--solver-name={} --fwph-stop-check-tol 0.01".format(solver_name))
+
 do_one("hydro", "hydro_cylinders.py", 3,
-       "--BFs 3 3 --bundles-per-rank=0 --max-iterations=100 "
+       "--branching-factors 3 3 --bundles-per-rank=0 --max-iterations=100 "
        "--default-rho=1 --with-xhatshuffle --with-lagrangian "
        "--solver-name={}".format(solver_name))
 do_one("hydro", "hydro_cylinders_pysp.py", 3,
-       "--BFs 3 3 --bundles-per-rank=0 --max-iterations=100 "
+       "--branching-factors 3 3 --bundles-per-rank=0 --max-iterations=100 "
```

Comment:
```suggestion
       "--bundles-per-rank=0 --max-iterations=100 "
```
`hydro_cylinders_pysp.py` gets its scenario tree from `PySPModel`